chapter_install/index_vn.md

```diff
@@ -127,7 +127,8 @@ proceed to :ref:`sec_gpu` for instructions
 to install a GPU-supported MXNet.
 -->
 
-*dịch đoạn phía trên*
+Trước khi cài đặt MXNet, hãy kiểm máy của bạn đã có card màn hình đúng chuẩn hay chưa (không phải những card màn hình hỗ trợ hiển thị trên các máy tính xách tay thông thường).
```

Comment:
```suggestion
Trước khi cài đặt MXNet, kiểm tra thiết bị của bạn xem có GPU (card màn hình) đúng chuẩn hay không (không phải những GPU tích hợp hỗ trợ hiển thị trên các máy tính xách tay thông thường).
```
Mình thấy nên thống nhất toàn bộ giữ nguyên GPU thay vì chuyển sang card màn hình.